cfgov/v1/models/molecules.py

```diff
@@ -12,121 +12,90 @@ def isRequired(field_name):
     heading = blocks.CharBlock(max_length=100, required=True)
-    content = blocks.RichTextBlock(blank=True)
-    links = blocks.ListBlock(blocks.StructBlock([
-        ('text', blocks.CharBlock(required=False)),
-        ('url', blocks.URLBlock(required=False)),
-    ], icon='user', required=False)
-    )
+    body = blocks.RichTextBlock(blank=True)
+    links = blocks.ListBlock(atoms.Hyperlink(), required=False)
 
     class Meta:
         icon = 'link'
-        template = 'v1/demo/molecules/half_width_link_blob.html'
+        template = '_includes/molecules/half-width-link-blob.html'
 
 
 class ImageText5050(blocks.StructBlock):
-    title = blocks.CharBlock(max_length=100, required=True)
-    description = blocks.RichTextBlock(blank=True)
-    image = ImageChooserBlock(required=False)
-    image_path = blocks.CharBlock(required=False)
-    image_alt = blocks.CharBlock(required=False)
+    heading = blocks.CharBlock(max_length=100, required=True)
+    body = blocks.RichTextBlock(blank=True)
+    image = atoms.ImageBasic()
     is_widescreen = blocks.BooleanBlock(required=False)
     is_button = blocks.BooleanBlock(required=False)
-    link_url = blocks.URLBlock(required=False)
-    link_text = blocks.CharBlock(max_length=100, required=False)
-
-    def clean(self, data):
-        error_dict = {}
-        try:
-            block_data = super(ImageText5050, self).clean(data)
-        except ValidationError as e:
-            error_dict.update(e.params)
-            block_data = data
-
-        if not block_data['image'] and not block_data['image_path'] and not block_data['image_alt']:
-            img_err = ['Please upload or enter an image path']
-            error_dict.update({'image': img_err, 'image_path': img_err, 'image_alt': isRequired('Image alt')})
-
-        if block_data['image'] and block_data['image_path']:
-            img_err = ['Please select one method of image rendering']
-            error_dict.update({
-                'image': img_err,
-                'image_path': img_err})
-
-        if block_data['image_path'] and not block_data['image_alt']:
-            error_dict.update({'image_alt': isRequired('Image Alt')})
-
-        if error_dict:
-            raise ValidationError("ImageText5050 validation errors", params=error_dict)
-        else:
-            return block_data
+    link = atoms.Hyperlink()
 
     class Meta:
         icon = 'image'
-        template = 'v1/demo/molecules/image_text_5050.html'
+        template = '_includes/molecules/image-text-50-50.html'
 
 
 class ImageText2575(blocks.StructBlock):
     heading = blocks.CharBlock(max_length=100, required=True)
     body = blocks.RichTextBlock(required=True)
-    image = ImageChooserBlock(required=False)
-    image_path = blocks.CharBlock(required=False)
-    image_alt = blocks.CharBlock(required=False)
-    link_url = blocks.URLBlock(required=False)
-    link_text = blocks.CharBlock(max_length=100, required=False)
+    image = atoms.ImageBasic()
```

Comment:
Can we drop the url field and leave only the Chooser and the Alt text field?

![image](https://cloud.githubusercontent.com/assets/1860176/11284485/1cbe144a-8ed8-11e5-806a-55bce24070aa.png)